mediapipe/calculators/geti/scripts/download_data.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+DATA_FOLDER="/data"
+REPOSITORY_PATH="https://repository.toolbox.iotg.sclab.intel.com/mediapipe"
```

Comment:
Are those models public? Can we point to the public repository?

Reply:
This will be changed to public repository in next step.